src/libs/ReimbursementAccountUtils.js

```diff
@@ -25,18 +27,18 @@ function getErrors(props) {
 
 /**
  * @param {Object} props
- * @param {String} inputKey
+ * @param {String} path
  */
-function clearError(props, inputKey) {
+function clearError(props, path) {
     const errors = getErrors(props);
-    if (!errors[inputKey]) {
-        // No error found for this inputKey
+    if (!lodashGet(errors, path, false)) {
+        // No error found for this path
         return;
     }
 
-    // Clear the existing error for this inputKey
-    const newErrors = {...errors};
-    delete newErrors[inputKey];
+    // Clear the existing errors
+    const newErrors = lodashCloneDeep(errors);
+    lodashSet(newErrors, path, null);
```

Comment:
NAB, but since we are resetting the errors is there a reason we should set to null instead of delete the key? I think maybe it would be marginally better to delete the key, but probably doesn't make a huge difference.

Reply:
You are right, I was setting it to `null` to make Onyx clear it, but we are already doing that inside `setBankAccountFormValidationErrors`.

Changed to use `unset` from lodash.